lib/image/eight-bit-rgba-to-png.js

```diff
@@ -0,0 +1,80 @@
+const {unsigned: crc32} = require('buffer-crc32');
+const zlib = require('node:zlib');
+const { PNG } = require('../constants');
+
+const COMPRESSION_LEVEL = 3;
+
+const convertRgbaToScanlines = (rgba, width, height) => {
+    const stride = width * PNG.RGBA_CHANNELS;
+    const scanlines = Buffer.allocUnsafe(height * (1 + stride)); // extra byte for filter
```

Comment:
"allocUnsafe" is faster, because it does not fill the buffer with zeroes, leaving trash data
thats fine, because we fill it all the way
And thats safe, because we throw, if for some reason buffer wasn't rewritten fully (which would not happen)